File: hw_24-08-21/app/src/main/java/ru/androidschool/sqliteexample/presentation/MainActivity.java
package ru.androidschool.sqliteexample.presentation;

import androidx.appcompat.app.AppCompatActivity;
import androidx.lifecycle.ViewModelProvider;
import androidx.recyclerview.widget.RecyclerView;

import android.os.Bundle;

import ru.androidschool.sqliteexample.R;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        RecyclerView listView = findViewById(R.id.bookList);

        MainViewModel viewModel = new ViewModelProvider(
                this, new BooksViewModelFactory(getApplicationContext())
        ).get(MainViewModel.class);
        viewModel.getBooksData().observe(this, books ->
                listView.setAdapter(new BooksAdapter(books)));
    }

}

File: hw_22-06-21_test_project/app/src/main/java/ru/totowka/homework_22_06_2021/MainActivity.kt
package ru.totowka.homework_22_06_2021

import android.os.Bundle
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    companion object {
        const val MESSAGE = "Hello, world"
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        Toast.makeText(this, MESSAGE, Toast.LENGTH_LONG).show()
    }
}